app/api/routes.py

```diff
@@ -430,6 +440,27 @@ def update_resource(id, json, db):
         return utils.standardize_response(status_code=500)
 
 
+def create_resources(json, db):
+    try:
+        if isinstance(json, dict):
+            return utils.standardize_response(payload=dict(
+                data=create_resource(json, db)))
+
+        created_resources_list = []
```

Comment:
`created_resources = [create_resource(resource, db) for resource in json]`

No need to use hungarian notation, and confuse `res` with resources or response.